test/conformance/test_suite_test.go

```diff
@@ -4,21 +4,21 @@
 package test_test
 
 import (
-	"context"
+	//"context"
```

Comment:
Is this no longer necessary?  If so, delete, don't comment out.

Reply:
fixed